tests/integration/ha_tests/test_rollback_to_master_label.py

```diff
@@ -133,7 +134,20 @@ async def test_fail_and_rollback(ops_test, continuous_writes) -> None:
 
     logger.info("Re-refresh the charm")
     await ops_test.juju(
-        "refresh", DATABASE_APP_NAME, "--switch", "postgresql-k8s", "--channel", "14/stable"
+        "download",
+        "postgresql-k8s",
+        "--revision",
+        str(LABEL_REVISION),
+        "--filepath",
+        f"/tmp/postgresql-k8s_r{LABEL_REVISION}.charm",
```

Comment:
Cannot use revision flag when refreshing from a local charm, so we have to download the exact revision and do another local refresh